docusaurus/docs/engineering/boba-backend/APIs/intro.md

```diff
@@ -30,70 +63,130 @@ Add a quick explanation of different type of request parameters (URL, query and
+- **Authentication needed:** An HTTP Status Code of `401` if creating the contribution requires authentication data and none was found in the request.
+- **Conflict:** An HTTP Status Code of `409` if a contribution with the given id already exists.
 
-:::note
+:::tip
 
-`POST` requests are most often used by browsers when submitting forms.
+`POST` requests are often used by browsers when submitting forms.
 
 :::
 
 ### PUT: Update a contribution in a thread
 
-The client wants to **update** a contribution with id `contribution_123` in the thread with id `thread_456`. To achieve this, the client sends a `PUT` request to the `/threads/thread_456/contributions/contribution_123` endpoint. The `PUT` request would also include the updated contribution data as a `payload`.
+The client wants to **update** a contribution with id `contribution_123` in the thread with id `thread_456`.
+
+To do this, the client sends the following request:
+
+- **Endpoint:** `/threads/thread_456/contributions/contribution_123`
+- **HTTP Method:** `PUT`
+- **Payload:** The content of the updated contribution in the format specified by the server.
 
 Possible responses include:
 
-- An HTTP Status Code of `201` if the contribution was successfully created. This might include a `payload` that contains the finalized contribution data.
-- An HTTP Status Code of `403` if the user is not authorized to update the contribution.
-- An HTTP Status Code of `404` if the contribution does not exist.
+- **Success:** An HTTP Status Code of `201` if the contribution was successfully updated. This might include a `payload` that contains the finalized contribution data.
+- **Authentication needed:** An HTTP Status Code of `401` if updating the contribution requires authentication data and none was found in the request.
+- **Invalid authentication:** An HTTP Status Code of `403` if the authenticated user is not authorized to update the contribution.
+- **Not found:** An HTTP Status Code of `404` if the contribution does not exist.
+
+:::tip
+
+`PUT` requests are not easily accessible through browsers! To issue one, you can
+use a tool like [Postman](docs/engineering/boba-backend/using-postman) or write a test for your endpoint.
+
+:::
 
 ### DELETE: Delete a contribution in a thread
 
-The client wants to **delete** a contribution with id `contribution_123` in the thread with id `thread_456`. To achieve this, the client sends a `DELETE` request to the `/threads/thread_456/contributions/contribution_123` endpoint.
+The client wants to **delete** a contribution with id `contribution_123` in the thread with id `thread_456`.
+
+To do this, the client sends the following request:
+
+- **Endpoint:** `/threads/thread_456/contributions/contribution_123`
+- **HTTP Method:** `DELETE`
+- **Payload:** None
 
 Possible responses include:
```

Comment:
Yeah, so same as the others, are the possible reponses listed an exhaustive list or are some left out?